### ISSUE TITLE
auto-restart?

### ISSUE THREAD

**Issue by quape** [bordalix/yamup#11]
Hi there,
today I rebooted server but app wasn't restartet. I had a look but didn't find `/etc/init.d/<appName>`. `upstart` also doesn't exist. What to do?

**Comment by bordalix** [comment]
Which Ubuntu version do you have? Please show me the output of `lsb_release -a`

Upstart is for older versions of Ubuntu (< 16), the preferred method for recent versions is to use `systemd`.

Having said that, with upstart the file created is `/etc/init/<appName>.conf` (ends with .conf, not .upstart).

**Comment by quape** [comment]
Ubuntu 18.04.3 LTS

**Comment by bordalix** [comment]
So you should be running systemd, and not upstart.

You should have a `/etc/systemd/system/<appName>.service` file.

And it should have something like this:
```
[Unit]
Description=<appName>

[Service]
ExecStart=/opt/<appName>/start.sh
Restart=always

StandardOutput=syslog
StandardError=syslog
SyslogIdentifier=<appName>

[Install]
WantedBy=multi-user.target
```

To start it, just run:
- `sudo service <appName> start` to start
- `sudo service <appName> stop` to stop

If this doesn't work for you, check if you have the last version of yamup installed:

`npm update -g yamup`

and then run a new `yamup setup` from your project directory.

**Comment by quape** [comment]
There is a `/etc/systemd/system/meteor.service` and I can run `service meteor start` which works fine but it isn't run automatically when system boots.

**Comment by bordalix** [comment]
This should do it:

`sudo service enable <appName>`

You only have to do it once.

I guess I should add it to yamup 😄

**Comment by quape** [comment]
Hey João, thanks, that works fine.
Adding it would be great, for next version :)
yamup is really a great tool btw!!

**Comment by bordalix** [comment]
I've just published yamup 0.8.0 which will enable your app to start after reboot.

Update to the last version via `npm update -g yamup`

If you find any error, please open a new issue, I'm closing this one.

Thank you for your help.
